build-info.json

```diff
@@ -1,6 +1,6 @@
 {
   "appVersion": "1.9.1",
-  "infraVersion": "3.19.1.+",
+  "infraVersion": "3.20.0.+",
```

Comment:
This file can be completely removed? If not it should update appVersion and infraVersion to latest (1.10.0 and 3.21.0+)